README.md

```diff
@@ -1,54 +1,314 @@
+
+[to top](https://github.com/TASVideos/BizHawk/blob/master/README.md#bizhawk)
+
+### GNU+Linux and macOS
+
+*Compiling* requires MSBuild and *running* requires Mono and WINE, but **BizHawk does not run under WINE** — only the bundled libraries are required.
+
+If you use GNU+Linux, there might be a `bizhawk-git` package or similar in the same repo as the main package. If it's available, installing it will automate the build process.
+
+Building is as easy as:
+```sh
+git clone https://github.com/TASVideos/BizHawk.git BizHawk_master && cd BizHawk_master
+# or ssh: git clone git@github.com:TASVideos/BizHawk.git BizHawk_master && cd BizHawk_master
+msbuild /p:Configuration=Release BizHawk.sln
+```
+
+Remove the `/p:...` flag from MSBuild if you want debugging symbols.
+
+If your distro isn't listed under *Installing* above, `libblip_buf` probably isn't in your package repos. You can easily [build it yourself](https://gitlab.com/TASVideos/libblip_buf/blob/unified/README.md).
+
+Once built, see [*Installing*](https://github.com/TASVideos/BizHawk/blob/master/README.md#gnulinux-and-macos) above, substituting the repo's `output` folder for the download.
+
+Again, if your distro isn't listed there, you might get an "Unknown distro" warning in the terminal, and BizHawk may not open or may show the missing dependencies dialog. You may need to add your distro to the case statement in the script, setting `libpath` to the location of `d3dx9_43.dll.so` (please do share if you get it working).
+
+[to top](https://github.com/TASVideos/BizHawk/blob/master/README.md#bizhawk)
+
+## Usage
+
+#### Loading firmware
+
+You may have seen a dialog saying "You are missing the needed firmware files [...]" when trying to open a rom. Pressing "Yes" opens the Firmware Manager, or you can go to `Config` > `Firmwares...`.
+
+To load firmwares, the easiest way is to click "Import" in the menubar, navigate to the dumped firmware(s), select them all, and click "Open". It's a good idea to have them copied into the `Firmware` folder, which is nicely organised, when prompted. If you were trying to open a rom, click "Close and reload ROM" to do that. Keep in mind some firmware is optional and some have multiple versions, only one of which needs to be set.
+
+#### Identifying a good rom
+
+With a core and game loaded, look in the very left of the status bar (`View` > `Display Status Bar`):
+* a green checkmark means you've loaded a "known good" rom;
+* a "???" (don't remember) means you've loaded a "known bad" rom, created by incorrect dumping methods; and
+* a ?-block means you've loaded something unknown, possibly a romhack or homebrew title.
+
+#### Rebinding keys and controllers
+
+TODO
+
+#### Changing cores
+
+To change which core is used for NES, SNES, GB, or GBA, go to `Config` > `Cores`. There, you'll also find the `GB in SGB` item, which is a checkbox that makes GB games run with the *Super Game Boy*, and `N64 Video Plugin Settings`, which you shouldn't touch.
```

Comment:
When you load some systems, there's also an option to change the core (for NES at least) available from the system menu (`NES` in that case). And of course one is free to touch N64 plugin settings (which is also available from the `N64` menu when you load that system): change resolution, plugin, experiment with plugin settings to get the best experience.

Reply:
I'm thinking it's easier where they're all together. Also I didn't finish that sentence, is it good as-is or should we go into more detail?

Reply:
Then just fix the part about n64 plugins.

Reply:
Have I?

Reply:
> And of course one is free to touch N64 plugin settings (which is also available from the N64 menu when you load that system): change resolution, plugin, experiment with plugin settings to get the best experience

> N64 Video Plugin Settings, which you shouldn't touch unless you know what you're doing.

Not at all.

Reply:
I agree, done